README.md

```diff
@@ -95,3 +95,17 @@ Helm 3 is now released and `tiller` is no longer required.
 We have added the `hello-helm3` examples here with updated instructions if you would like to use the latest version of Helm.
 
 You can read more about the changes between version 2 and 3 [here](https://v3.helm.sh/docs/faq/#changes-since-helm-2).
+
+### `kubectl run` and Pod/Deployment v1.18 change
+
+In the 1.18 release of Kubernetes the `kubectl run` command changed from creating a `Deployment` by default to creating a `Pod` instead.
+
+We have a few examples where we use `kubectl run` to get familiar with running a container in k8s.
+
+Later we to discuss why using the declarative `kubectl apply -f...` is preferred rather than using commands like `create`, `edit` or `run` directly because those changes are not captured in source control.
```

Comment:
```suggestion
Later we discuss why using the declarative `kubectl apply -f...` is preferred over the imperative `create`, `edit` or `run`, because your version-controlled YAML files always reflect the real state of the cluster.
```